multihost_test/Bugzillas/test_Regression.py

```diff
@@ -464,3 +471,73 @@ def test_bz450262(self, multihost, create_backup):
         client.run_command(f"rm -f {output_file}")
         client.run_command(f"groupdel {second_group}")
         client.run_command(f"groupdel {first_group}")
+
+    @pytest.mark.tier1
+    def test_bz1114081_1285547(self, multihost, create_backup):
+        """
+        :title:pam_lastlog unable to reset locked account
+        :id: 94626b58-f7c6-11ee-bf95-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1285547
+                   https://bugzilla.redhat.com/show_bug.cgi?id=1114081
+        :steps:
+          1. Creates a new user test1.
+          2. Clear the contents of the lastlog file and sets its context using restorecon.
+          3. Generates options for the lastlog command and checks if they are all present.
+          4. Simulates a login for user test1 that is five days in the past using log_back_in_time.
+          5. Checks the last login entry for test1 to ensure it's not showing "Never logged in".
+          6. Lastlog command is run to check if the user (represented by the {user} variable)
+            has logged in before 4 days ago.
+          7. Check if an exception is raised when attempting to query login
+            information before 6 days ago for the specified user.
```

Comment:
Change by `Run lastlog to check if the user has logged in before the last 6 days.`